pkg/client/src/app/pages/repositories/Mvn.tsx

```diff
@@ -164,8 +188,7 @@ export const RepositoriesMvn: React.FunctionComponent = () => {
                   isInline
```

Comment:
While we're in here can we remove this `isInline` prop on the clear button? It'll get a natural padding that prevents it from being right up against the text field like it is now.